Add pulumi exports for RDS instances

twunderlich-grapl opened the pull request:
<!-- Thank you for your contribution to Grapl! Please take some time
to fill out this short template to help us review and understand your
PR. -->

### Which issue does this PR correspond to?
https://github.com/grapl-security/issue-tracker/issues/833
<!-- Please provide a link to the GitHub Issue this pull request is
meant to address. -->

### What changes does this PR make to Grapl? Why?
This adds pulumi exports to the AWS section for the rds instances. These exports are relied upon by the integration tests pulumi project, which is causing the Buildkite pipeline to fail.
<!-- Please describe the functional impact of these changes and the
rationale behind them. This will help us understand your approach to
solving the problem. -->

### How were these changes tested?
pulumi up in the grapl project, followed by pulumi up in the integration-tests project
<!-- Please describe how these changes were tested. There should be
sufficient detail that reviewers can replicate your testing
procedure. If the procedure is a manual one that's OK, your
description will help us work with you to get it automated. -->

Commit:
Add pulumi exports for RDS instances

While we currently have exports for these in the local grapl we forgot
to export these when running in AWS. This means that the CI pipeline
and AWS sandboxes can't run integration tests

christophermaier commented on pulumi/grapl/__main__.py:
```diff
@@ -387,6 +387,34 @@ def main() -> None:
             nomad_agent_security_group_id=nomad_agent_security_group_id,
         )
 
+        pulumi.export(
```
Are we exporting everything else in this project as single key/value pairs, or consolidating into objects? That is, export an object for, say, "plugin-registry-db" that has "hostname", "port", "username", etc. keys?

twunderlich-grapl replied:
Well as much as I'd like to do that, the reason why I'm not is because the local postgres Database object doesn't support these attributes.  We could create a plain old python object, but that feels like the same amount of work as just passing in each attribute

Commit:
Export objects instead of multiple strings

Instead of enumerating every individual attribute we care about we
should just pass through the object to other pulumi projects.

Commit:
Revert "Export objects instead of multiple strings"

This reverts commit f3b1b68bcb31092b214416c1c7212296768bd3c4.

So the reason why we're exporting attributes instead of shipping an object
is because the local postgres object doesn't support adding arbitrary
attributes and also doesn't have most of these attributes in the instance
object itself. :/
